TicTacToe_iOS/Views/ContentView.swift

```diff
@@ -28,39 +28,21 @@ struct ContentView: View {
                         Button(action: {
                             print("Top Left Button!")
                             game.TopLeft()
-                            noMove = game.notEmpty
-                            xWins = game.winnerX
-                            oWins = game.winnerO
-                            itsDraw = game.draw
                         }) {
                             Text (game.topLeft)
                                 .bold()
                                 .frame(maxWidth: .infinity, maxHeight: .infinity)
                                 .font(.largeTitle)
                                 .foregroundColor(.red)
                         }
-                        .alert(isPresented: $noMove, content: {
-                            return Alert(title: Text("Oooops!"), message: Text ("This spot is already taken buddy"), dismissButton: .default(Text("Try Again !")))
-                        })
-                        .alert(isPresented: $xWins, content: {
-                            return Alert(title: Text("Congratulations !!!"), message: Text ("Player - X - has won"), dismissButton: .default(Text("Play Again")))
-                        })
-                        .alert(isPresented: $oWins, content: {
-                            return Alert(title: Text("Congratulations !!!"), message: Text ("Player - O - has won"), dismissButton: .default(Text("Play Again")))
+                        .alert(isPresented: $game.alert, content: {
```

Comment:
You might want to change the remaining alerts to be like this one. I don't know why its not reflecting here but I think I did change them